TestStack.Dossier/TestDataBuilder.cs

```diff
@@ -71,10 +72,22 @@ public TObject Build()
         }
 
         /// <summary>
-        /// Build the actual object - override this and call the constructor and any other methods.
+        /// Build the actual object - you can optionally override this and call the constructor and any other methods.
         /// </summary>
         /// <returns>The built object</returns>
-        protected abstract TObject BuildObject();
+        protected virtual TObject BuildObject()
+        {
+            var model = Any.Fixture.Create<TObject>();
+
+            var properties = Reflector.GetSettablePropertiesFor<TObject>();
```

Comment:
I wonder if we should only set public property setters from the `Builder<T>` class to encourage use of that class for DTOs and the default one for DDD entities?